lib/DimmableLightDevice.js

```diff
@@ -1,15 +1,24 @@
 'use strict';
 
-const util = require('/node_modules/homey-meshdriver/lib/util');
+const util = require('homey-meshdriver/lib/util');
+
+const ZigBeeLightDevice = require('homey-meshdriver').ZigBeeLightDevice;
+
+class DimmableDevice extends ZigBeeLightDevice {
 
-module.exports = {
 	onFlowActionSetDimWithCustomTransitionTime(dimtype, dimwaarde, transitietijd) {
-		this.log('type_to_dim', parseInt(dimtype, 0));
+		dimtype = parseInt(dimtype, 10);
```

Comment:
moved parsing of `dimtype` to the top. Also changed the `base` parameter from `0` to `10` since the value should be parsed in base-10